server/proxy/proxy.go

```diff
@@ -286,6 +288,7 @@ func (p *Proxy) NotifyRemoteWebhook(app *apps.App, data []byte, webhookPath stri
 		},
 		Context: p.conf.Get().SetContextDefaultsForApp(app.AppID, &apps.Context{
 			ActingUserID: app.BotUserID,
+			Locale:       utils.GetLocale(p.conf.MattermostAPI(), p.conf.MattermostConfig().Config(), app.BotUserID),
```

Comment:
What is the use case for a bot specific locale?

Reply:
You are right, we should probably use the server locale instead of the bot.